partner_financial_risk/tests/test_partner_financial_risk.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+# © 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.account.tests.account_test_classes import AccountingTestCase
+from odoo import fields
+
+
+class TestPartnerFinancialRisk(AccountingTestCase):
```

Comment:
We chose to depend on AccountingTestCase because there was a bug with the traditional one. The tests were asking to install a chart of account:
```
2017-06-28 20:08:48,434 13918 INFO MaisonDF-Tests odoo.modules.module: odoo.addons.partner_sale_risk.tests.test_partner_sale_risk running tests.
2017-06-28 20:08:49,840 13918 INFO MaisonDF-Tests odoo.addons.partner_sale_risk.tests.test_partner_sale_risk: test_invoice_amount (odoo.addons.partner_sale_risk.tests.test_partner_sale_risk.TestPartnerSaleRisk)
2017-06-28 20:08:51,074 13918 ERROR MaisonDF-Tests odoo.addons.partner_sale_risk.tests.test_partner_sale_risk: ERROR
2017-06-28 20:08:51,081 13918 INFO MaisonDF-Tests odoo.addons.partner_sale_risk.tests.test_partner_sale_risk: test_sale_order (odoo.addons.partner_sale_risk.tests.test_partner_sale_risk.TestPartnerSaleRisk)
2017-06-28 20:08:52,524 13918 INFO MaisonDF-Tests odoo.addons.partner_sale_risk.tests.test_partner_sale_risk: ======================================================================
2017-06-28 20:08:52,524 13918 ERROR MaisonDF-Tests odoo.addons.partner_sale_risk.tests.test_partner_sale_risk: ERROR: test_invoice_amount (odoo.addons.partner_sale_risk.tests.test_partner_sale_risk.TestPartnerSaleRisk)
2017-06-28 20:08:52,524 13918 ERROR MaisonDF-Tests odoo.addons.partner_sale_risk.tests.test_partner_sale_risk: Traceback (most recent call last):
2017-06-28 20:08:52,525 13918 ERROR MaisonDF-Tests odoo.addons.partner_sale_risk.tests.test_partner_sale_risk: `   File "/var/lib/jenkins/workspace/MaisonDF-Tests/git/partner-contact/partner_sale_risk/tests/test_partner_sale_risk.py", line 69, in test_invoice_amount
2017-06-28 20:08:52,525 13918 ERROR MaisonDF-Tests odoo.addons.partner_sale_risk.tests.test_partner_sale_risk: `     wizard.create_invoices()
2017-06-28 20:08:52,525 13918 ERROR MaisonDF-Tests odoo.addons.partner_sale_risk.tests.test_partner_sale_risk: `   File "/var/lib/jenkins/workspace/MaisonDF-Tests/git/odoo/addons/sale/wizard/sale_make_invoice_advance.py", line 164, in create_invoices
2017-06-28 20:08:52,525 13918 ERROR MaisonDF-Tests odoo.addons.partner_sale_risk.tests.test_partner_sale_risk: `     self._create_invoice(order, so_line, amount)
2017-06-28 20:08:52,526 13918 ERROR MaisonDF-Tests odoo.addons.partner_sale_risk.tests.test_partner_sale_risk: `   File "/var/lib/jenkins/workspace/MaisonDF-Tests/git/odoo/addons/sale/wizard/sale_make_invoice_advance.py", line 75, in _create_invoice
2017-06-28 20:08:52,526 13918 ERROR MaisonDF-Tests odoo.addons.partner_sale_risk.tests.test_partner_sale_risk: `     (self.product_id.name,))
2017-06-28 20:08:52,526 13918 ERROR MaisonDF-Tests odoo.addons.partner_sale_risk.tests.test_partner_sale_risk: ` UserError: (u'There is no income account defined for this product: "Down payment". You may have to install a chart of account from Accounting app, settings menu.', None)
2017-06-28 20:08:52,526 13918 INFO MaisonDF-Tests odoo.addons.partner_sale_risk.tests.test_partner_sale_risk: Ran 2 tests in 4.089s
2017-06-28 20:08:52,527 13918 ERROR MaisonDF-Tests odoo.addons.partner_sale_risk.tests.test_partner_sale_risk: FAILED
2017-06-28 20:08:52,527 13918 INFO MaisonDF-Tests odoo.addons.partner_sale_risk.tests.test_partner_sale_risk:  (errors=1)
2017-06-28 20:08:52,527 13918 ERROR MaisonDF-Tests odoo.modules.module: Module partner_sale_risk: 0 failures, 1 errors
```
The problem was solved by depending on AccountingTestCase. And it worked with our Jenkins, so I won't have time to find another solution.